static-site/app/staging/[[...staging]]/validateStagingUrl.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { notFound, useParams }from "next/navigation";
+
+/**
+  * A React Client Component to detect when an invalid `/staging/` URL
+  * is requested, which calls the `notFound()` method to redirect to
+  * the `not-found.tsx` component
+  *
+  * Note that any actually valid `/staging/<something>` URL will be
+  * redirected at the Express router level, before the Next.js router
+  * is engaged, so if we are trying to render a `/staging/<something>`
+  * URL, it _is_ an error
+ */
+export default function ValidateStagingUrl(): null {
+  const params = useParams();
+
+  if (params && params["staging"]) {
+    notFound();
+  }
```

Comment:
I wonder if another name would be more informative here. Maybe `staging/[[...slug]]/…` to match [Next.js docs](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments)? Same would go for `pathogens/[[...slug]]`. `blog/[id]` is meaningful as-is.


```suggestion
  if (params && params["slug"]) {
    notFound();
  }
```

Not the fault of this PR since the old implementation [uses [[...staging]]](https://github.com/nextstrain/nextstrain.org/blob/db66c00f9ca610978ec4df6e46570b9858a33611/static-site/pages/staging/%5B%5B...staging%5D%5D.jsx)

Reply:
Opened #1137 for this.